image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -253,14 +253,6 @@
   {{ $_ := set $._rox.scanner "disable" true }}
 [< end >]
 
-{{/* Disable local scanner on non-OpenShift clusters by default. */}}
```

Comment:
During helm installation a user would still need to set `scanner.disable=false` in order for the scanner slim to be installed (by default it will NOT be installed on *k8s (or OCP) for helm installs)

The `ROX_FORCE_LOCAL_IMAGE_SCANNING` env var will not exist in the next release (4.1), enabling/disabling local scanning will instead be config driven via the API / UI and only clusters that have a local scanner will be able to be selected in that case.

Do you think relying on `scanner.disable` would be OK?

Reply:
@dcaravel No, this would not be sufficient as long as the customer can install local scanners into unsupported clusters which break the installation.